drones_controller/DroneWaypoint[] waypoints
float32 waypointWaitSeconds
bool isPathCyclic
